src/Integration.Vsix/CFamily/IProcessRunner.cs

```diff
@@ -25,6 +25,6 @@ namespace SonarLint.VisualStudio.Integration.Vsix.CFamily
 {
     public interface IProcessRunner
     {
-        bool Execute(ProcessRunnerArguments runnerArgs);
+        void Execute(ProcessRunnerArguments runnerArgs);
```

Comment:
There's a comment in line 21 saying this was copied from the Scanner for MSBuild, which suggests that the implementation is the same. Since this isn't the case, the comment needs to be updated or removed.
Same for the implementation class.